airflow/config_templates/default_airflow.cfg

```diff
@@ -243,6 +243,7 @@ default_cpus = 1
 default_ram = 512
 default_disk = 512
 default_gpus = 0
+allow_illegal_arguments = True
```

Comment:
Shouldn't be here a short note explaining what this setting does?
Something like
`# When set to False raise exception on unused operator arguments`

Reply:
I added in the seperate PR.
https://github.com/apache/airflow/pull/6435